extensions/PhabBugz/lib/WebService.pm

```diff
@@ -40,6 +41,8 @@ use constant PUBLIC_METHODS => qw(
     revision
 );
 
+use constant PHAB_BMO_USER_EMAIL => 'phab-bot@bmo.tld';
```

Comment:
In the future, we should put this within the PhabBugz config, but to keep this PR simple and merged quickly let's roll with this.  This is a semi-blocker of an important bug validation piece